apollo-federation/src/sources/connect/snapshots/apollo_federation__sources__connect__url_template__test_parse__url_path_template_parse-3.snap

```diff
@@ -35,9 +43,17 @@ Ok(
             }: Component {
                 parts: [
                     Var(
-                        Variable {
-                            var_type: Config,
-                            path: "b",
+                        VariableReference {
+                            namespace: VariableNamespace {
+                                namespace: $config,
+                                location: 21..28,
+                            },
+                            path: [
+                                VariablePathPart {
+                                    part: "b",
+                                    location: 29..30,
```

Comment:
I'm not scrutinizing all these locations closely—but just to double check, did you verify that the highlight locations via LSP look correct for paths after the namespace? Easy to get off by ones

Reply:
I've done a bunch of testing in VS Code manually, and added cases to these snapshot tests, exactly for this reason (and did find and fix a few bugs as a result).